### DISCUSSION TITLE
TBEAM V1.1 SX1262 fails init w -705 (RADIOLIB_ERR_SPI_CMD_TIMEOUT)

### DISCUSSION THREAD

**Discussion by MrSniffer** [jgromes/RadioLib#580] [Q&A]
I am trying to run code on the LilyGo TBeam V1.1 w SX1262 flavor. The begin call fails consistently with code -705 (SPI Cmd Timeout). I get this with my code and using the SX126x_Settings example provided with RadioLib.
 Here is the verbose debug output from the example sketch.

[SX1262] Initializing ... M	SX126x
failed, code -705

Not much there!

**Comment by jgromes** [comment]
Failing straight away with timeout is on SX126x most likely caused by incorrect/bad BUSY pin.

**Reply by MrSniffer** [reply]
Are the default values in the instantiation supposed to be the GPIO pins in the vendors board?
// SX1262 has the following connections:
// NSS pin:   10
// DIO1 pin:  2
// NRST pin:  3
// BUSY pin:  9
SX1262 radio1 = new Module(10, 2, 3, 9);
For this board, I know
  #define NSS 18
  #define NRST 23
  #define DIO0 26
  #define DIO1 13
but what is BUSY?????? I don't see it labeled on the pinouts.

**Reply by jgromes** [reply]
The default pins are for my own project (RadioShield), you will have to update it to match your board. Are you sure it's SX1262 and not SX127x? That one doesn't have the BUSY pin - and SX126x doesn't have a DIO0.

**Reply by MrSniffer** [reply]
For sure there is a LilyGo chip labelled SX1262! And I know to use DIO1. But I don't have an updated pinout or schematic for this flavor of the TBEAM. How could I locate a BUSY? And meanwhile, should I be using
begin(18,13,23,???) based on the GPIOs above?

**Reply by MrSniffer** [reply]
Your responses triggered a more productive search, and I found the answers here in:
https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/blob/master/examples/RadioLibExamples/SX1262/SX1262_Receive_Interrupt/utilities.h
NSS 18; DIO1 33; RST 23; BUSY 32
for anyone else wondering what they should be!

**Reply by aaseagrass** [reply]
Hi, sorry to interrupt, but I'm having same problem. I got failed initialization of my LoRa-E5 module which uses SX126x chip. I sued the SX126x code here. Could help me?
